config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
```

Comment:
This file should be ignored, no?

Reply:
I will clean later if you do not mind

Reply:
> This file should be ignored, no?

Regardless of being ignored or not, we already have it inside examples so imo we should do like for database.yml that it is only inside examples but inside config directly is in .gitignore so everybody can customize it was we wish.

> I will clean later if you do not mind

Sure, no problem :+1: